### PR TITLE
chore: added .codeclimate,yml for codeclimate run

### LINKED ISSUE #57: Add a .codeclimate.yml file to this repo
It is possible to configure how CodeClimate looks at a repository.
[These Docs](https://docs.codeclimate.com/docs/getting-started-configuration) explain how this is done. Please create an appropriate .codeclimate.yml for this repo. It should test the main language of the repo and run tests.

### PR DESCRIPTION
**Pull request by skshelar**
added .codeclimate,yml for codeclimate run

@mbernier edit resolves #57

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/java-http-client?pullRequest=58) <br/>All committers have signed the CLA.

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# to turn off an engine, set enabled to `false` or remove it
+#  rubocop:
+#    enabled: true
+#  golint:
+#    enabled: true
+#  gofmt:
+#    enabled: true
+#  eslint:
+#    enabled: true
+#  csslint:
```
Comment (mbernier) [inline .codeclimate.yml:26]: Can you please add SonarJava?

https://docs.codeclimate.com/v1.0/docs/sonar-java

**Review by mbernier — CHANGES_REQUESTED** [review]

File: .codeclimate.yml (hunk 1 above)
Comment (skshelar) [reply .codeclimate.yml:26]: Hi , I have added SonarJava.

**Review by mbernier — APPROVED** [review]

**Comment by mbernier** [comment]
Thank you!!

**Comment by thinkingserious** [comment]
Hello @skshelar,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**